spring-data-cassandra/src/test/java/org/springframework/data/cassandra/core/UpdateOptionsUnitTests.java

```diff
@@ -30,13 +33,16 @@
 
 	@Test // DATACASS-250
 	public void shouldConfigureUpdateOptions() {
+		Instant now = LocalDateTime.now().toInstant(ZoneOffset.UTC);
 
 		UpdateOptions updateOptions = UpdateOptions.builder()
 				.ttl(10)
+				.timestamp(now)
 				.withIfExists()
 				.build();
 
 		assertThat(updateOptions.getTtl()).isEqualTo(Duration.ofSeconds(10));
+		assertThat(updateOptions.getTimestamp() == now.toEpochMilli() * 1000);
```

Comment:
did you mean assertThat(mutated.getTimestamp()).isEqualTo(now.toEpochMilli() * 1000)?